README.md

```diff
@@ -41,6 +41,23 @@ provider:
     MYSECRET: /path/to/ssm/secret
 ```
 
+Alternatively, if you would like to reference the secrets by their name in the
+parameter store you can specify the secrets as a list of strings.
+
+```
+provider:
+  environmentSecrets:
+    - MYSECRET
+```
+
+Which is equivalent to:
+
+```
+provider:
+  environmentSecrets:
+    MY_SECRET: MY_SECRET
+```
+
 The plugin will create a json file with all the secrets. In the above example the ciphertext and ARN of the secret located at `path/to/ssm/secret` will be stored in the file under the key `MYSECRET`.
```

Comment:
This paragraph references `path/to/ssm/secret` which doesn't match up with the last example.  

What do you all think of having a basic getting started that just uses the 'simple' way, and then create a new `Advanced Configuration` section that details how to customize the key name.  This we we lower the barrier to adoption w/ as little upfront customization as possible, but then provide opportunities for 'advanced' config for power users.